src/sentry/features/helpers.py

```diff
@@ -1,18 +1,24 @@
 from typing import Any, Sequence
 
+from rest_framework.request import Request
 from rest_framework.response import Response
 
 from sentry import features
 from sentry.models import Organization
 
+# TODO(mgaeta): It's not currently possible to make a Callable with kwargs.
```

Comment:
See https://mypy.readthedocs.io/en/stable/kinds_of_types.html#callable-types-and-lambdas.

I'm not sure what exactly you're trying to do here, but something like `Callable[..., T]` is possible if you want to specify a function that takes arbitrary arguments and returns T.